### PR TITLE
Re-layout readme, add Simplified Chinese version readme

### PR DESCRIPTION
**Pull request by chencu5958**

### CODE REVIEW

**Comment by Syrent** [comment]
Nice changes, Thanks!

File: README.md [hunk 1]
```diff
@@ -1,28 +1,169 @@
...
+<th width="2000" colspan="2">
+</th>
+<h2>🧭 Links</h2>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td width="80" align="center" valign="top">
+    <br>
+    <a href="https://github.com/Syrent/VelocityReport/wiki"><img src="./blob/images/icon/bookmark.svg"></a>
```
Comment (Syrent) [inline README.md:30]: Missing image

File: README.md [hunk 2]
```diff
@@ -1,28 +1,169 @@
...
+    <p>
+      Don't know how to use it? <a href="https://github.com/Syrent/VelocityReport/wiki">Click here to start getting started</a>!
+      <br>
+      <a href="https://github.com/Syrent/VelocityReport/wiki/Why-VelocityReport">And why Velocity Reoprt</a>?
+    </p>
+  </td>
+</tr>
+  <td width="80" align="center" valign="top">
+    <br>
+    <a href="https://discord.gg/VZk2XU3kFg"><img src="./blob/images/icon/support.svg"></a>
```
Comment (Syrent) [inline README.md:43]: Missing image

File: README.md [hunk 3]
```diff
@@ -1,28 +1,169 @@
...
+    <h3>Support</h3>
+    <p>
+      Join our <a href="https://discord.gg/VZk2XU3kFg">discord community</a> for more help.
+    </p>
+  </td>
+</tr>
+<tr>
+  <td width="80" align="center" valign="top">
+    <br>
+    <a href="https://github.com/Syrent/VelocityReport/issues"><img src="./blob/images/icon/bug.svg"></a>
```
Comment (Syrent) [inline README.md:55]: Missing image

Comment (chencu5958) [reply README.md:55]: When I edit locally with vscode it displays correctly, maybe it's a path issue that's preventing it from displaying correctly

**Pull request merged**